### DISCUSSION TITLE
Implement Raycaster for many moving objects

### DISCUSSION THREAD

**Discussion by gonzalesMK** [isaac-sim/IsaacLab#1503] [Ideas]
I wish the following features on the `Raycaster` sensor:
  1. Handle more than one item on `mesh_prim_paths`
  2. Handle regex string on `mesh_prim_paths`
  3. Handle moving objects (but not deformable nor articulated) on `mesh_prim_paths`

I'm interested in working on that if it is ok.

**Comment by RandomOakForest** [comment]
Thank you for posting this and your interest on improving Isaac Lab. I will forward this to the team and write back as soon as possible.
